themes/classic/templates/catalog/category.tpl

```diff
@@ -16,7 +16,7 @@
     {/block}
 
     {block name='category_products'}
-      {include file='catalog/_partials/products.tpl' products=$products}
+      {include file='catalog/_partials/products.tpl' products=$products label=$category.name}
```

Comment:
I prefer `categoryName` instead of `label` it's easier to understand where this value come from